### DISCUSSION TITLE
Label stars using SAO numbers?

### DISCUSSION THREAD

**Discussion by GerritErdt** [Stellarium/stellarium#3227] [General]
Hi guys, 

is it possible to list the stars in Stellarium using their SAO numbers instead of their names? Skywatcher mounts mostly use SAO numbers, so it would be easier to just see these numbers on the map instead of names. 

CS Gerrit

**Comment by GerritErdt** [comment]
bump ;)

**Comment by alex-w** [comment]
By historical reasons Stellarium uses HIP as primary designation for bright stars and in recent years we added few (HR, HD, SAO, WDS) designations for cross-identified subset of related catalogs. Plus Bayer, Flamsteed and various designations for double & variable stars. So, we cannot show only SAO numbers as screen labels.

**Reply by GerritErdt** [reply]
Alright, thanks for your response. 

And I guess there is no way to change the default catalog that is used to label stars on the sky map, isn't it?

CS Gerrit

**Reply by alex-w** [reply]
Yes, you're right

**Reply by GerritErdt** [reply]
Ok, thanks for the clarification.
